src/Hosting/SystemdServices/src/ISystemdNotifier.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Hosting.SystemdServices
+{
+    /// <summary>
+    /// Provides support to notify systemd about the service status.
+    /// </summary>
+    public interface ISystemdNotifier
+    {
+        /// <summary>
+        /// Sends a notification to systemd.
+        /// </summary>
+        void Notify(ServiceState state, params ServiceState[] states);
```

Comment:
I can simplify this to accept a single state.

Reply:
The single state makes sense to me.